Resources/config/provider-resource.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="cmf_routing.resource.provider.class">Symfony\Cmf\Bundle\RoutingBundle\Resource\ResourceProvider</parameter>
+    </parameters>
+
+    <services>
+
+        <service id="cmf_routing.route_provider" class="%cmf_routing.resource.provider.class%">
```

Comment:
agreed. want to do that in a separate PR to keep it apart? that would also allow to mix orm and phpcr routing - not necessarily a good idea but there sure is one usecase for that somewhere.

it will be a micro-bc break if somebody has a compiler pass to change the route provider service, as he will then hit the alias instead of the service definition, but depending on the service will still look the same.